tac/guidelines/project-incubation-entry-considerations.md

```diff
@@ -8,7 +8,7 @@ nav_order: 5
 [//]: # (SPDX-License-Identifier: CC-BY-4.0)
 
 # Project Incubation Entry Considerations
-This document provides information to help people who are considering bringing their projects to LF Decentralized Trust (hereafter "LFDT") for [Incubation](../governing-documents/project-lifecycle#incubation). Specifically, it will outline items that the TAC will take into consideration when evaluating the project, as well as, some best practices that have been followed by other projects prior to the project proposal being submitted to the TAC. Our hope is that this document will smooth the entry process for new projects being proposed. The [project proposal process](../governing-documents/project-lifecycle#proposal) and the [template for project proposals](https://hyperledger.github.io/hyperledger-hip/) are outlined outside of this document. The goal of Incubation within LFDT is to provide a space for projects that have high potential to grow in the community. Ideas should start in [Labs](https://labs.hyperledger.org/).
+This document provides information to help people who are considering bringing their projects to LF Decentralized Trust (hereafter "LFDT") for [Incubation](../governing-documents/project-lifecycle#incubation). Specifically, it will outline items that the TAC will take into consideration when evaluating the project, as well as, some best practices that have been followed by other projects prior to the project proposal being submitted to the TAC. Our hope is that this document will smooth the entry process for new projects being proposed. The [project proposal process](../governing-documents/project-lifecycle#proposal) and the [template for project proposals](https://lf-decentralized-trust.github.io/project-proposals/) are outlined outside of this document. The goal of Incubation within LFDT is to provide a space for projects that have high potential to grow in the community. Ideas should start in [Labs](https://labs.hyperledger.org/).
```

Comment:
the Labs URL might also need to be updated pending the discussions on the TOC agenda for 10/31/2024